src/main/java/Todo.java

```diff
@@ -0,0 +1,15 @@
+public class Todo extends Task {
+
+    protected String by;
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+//        return "[T]" + super.toString().replaceFirst("[]", "");
+//        should use regular expression to avoid magic number, but haven't figured out how to do
+        return "[T]" + super.toString().substring(3);
```

Comment:
Can use constant to replace magic number

Reply:
Revised! Thx!